### PR TITLE
Update copyright notices for 2019 #trivial

### PR DESCRIPTION
**Pull request by djbe**
Do note that for consistency & fairness, I've removed people's names from the file headers. They were pretty much obsolete anyway, because multiple of people have worked on the same file, and we keep track of credits using the changelog.

The file headers are now also checked using SwiftLint. I did try using nested configurations to ensure the correct file header is used depending on the module/location, but there's a bug in SwiftLint that doesn't allow us to do that (yet): https://github.com/realm/SwiftLint/pull/1745#issuecomment-320802347

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

Seems like everything is in order 👍 You did a good job here! 🤝

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -766,9 +766,9 @@ let bananas = L10n.bananasOwner(5, "Olivier")
 
 ---
 
-# License
+# Licence
 
-This code and tool is under the MIT License. See the `LICENSE` file in this repository.
+This code and tool is under the MIT Licence. See the `LICENSE` file in this repository.
```
Comment (AliSoftware) [inline README.md]: I'm all for consistency between US/UK spelling, but then the LICENSE file should then be renamed. Or we should choose the License spelling. But either way be consistent 😉

File: Tests/Fixtures/Generated/JSON/inline-swift3-context-all-publicAccess.swift [hunk 2]
```diff
@@ -1,4 +1,5 @@
-// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
```
Comment (AliSoftware) [inline Tests/Fixtures/Generated/JSON/inline-swift3-context-all-publicAccess.swift]: How come we didn't have the `swiftlint:disable all` line on this file (and some others apparently) before this PR?

File: README.md (hunk 1 above)
Comment (djbe) [reply README.md]: Yeah I noticed that too, but I'm not sure if GitHub supports it? I'll give it a try and rename it 🤞

File: Tests/Fixtures/Generated/JSON/inline-swift3-context-all-publicAccess.swift (hunk 2 above)
Comment (djbe) [reply Tests/Fixtures/Generated/JSON/inline-swift3-context-all-publicAccess.swift]: Probablby 2 PRs that were developed in parallel and a needed change that got missed 🤷‍♂️

File: README.md (hunk 1 above)
Comment (AliSoftware) [reply README.md]: Mmh no match for `LICENCE` in https://help.github.com/articles/licensing-a-repository/🤔

Comment (djbe) [reply README.md]: But it is mentioned in https://github.com/benbalter/licensee:
https://github.com/benbalter/licensee/blob/489bc5e1ade015981dffbb1069ec4b9579a96e57/spec/licensee/project_files/license_file_spec.rb#L59

**Pull request merged**